port/zephyr/Kconfig

```diff
@@ -74,4 +74,6 @@ config GOLIOTH_ZEPHYR_THREAD_STACK_SIZE
 	help
 	  Threads stacks sizes for the use in golioth_sys_thread_create().
 
+rsource "../../examples/zephyr/common/Kconfig"
+
```

Comment:
`examples/zephyr/common/Kconfig` is already sourced by `examples/zephyr/golioth_basics/Kconfig`. Let's not source the same file twice.

Reply:
To me it makes more sense to have this available without needing to source it in the Kconfig file of every project that wants to use it. So I've taken the source directive out of the `examples/zephyr/golioth_basics/Kconfig` file.